e2e/vmss.go

```diff
@@ -338,6 +517,21 @@ func getBaseVMSSModel(name, sshPublicKey, customData, cseCmd string, cluster *Cl
 			},
 		}
 	}
+	if s.VHD.Windows() {
+		model.Identity = &armcompute.VirtualMachineScaleSetIdentity{
+			Type: to.Ptr(armcompute.ResourceIdentityTypeSystemAssignedUserAssigned),
+			UserAssignedIdentities: map[string]*armcompute.UserAssignedIdentitiesValue{
+				config.Config.VMIdentityResourceID(): {},
+			},
+		}
+		model.Properties.VirtualMachineProfile.StorageProfile.OSDisk.OSType = to.Ptr(armcompute.OperatingSystemTypesWindows)
+		model.Properties.VirtualMachineProfile.OSProfile.LinuxConfiguration = nil
+		model.Properties.VirtualMachineProfile.ExtensionProfile.Extensions[0].Properties.Publisher = to.Ptr("Microsoft.Compute")
+		model.Properties.VirtualMachineProfile.ExtensionProfile.Extensions[0].Properties.Type = to.Ptr("CustomScriptExtension")
+		model.Properties.VirtualMachineProfile.ExtensionProfile.Extensions[0].Properties.TypeHandlerVersion = to.Ptr("1.10")
+		model.Properties.VirtualMachineProfile.OSProfile.AdminUsername = to.Ptr("azureuser")
+		model.Properties.VirtualMachineProfile.OSProfile.AdminPassword = to.Ptr("pwnedPassword123!")
```

Comment:
Do we need to store password as a secret variable and avoid exposing it in public for securtiy?

Reply:
I wouldn't expect the password is enough to connect to the Windows VM. It's not the case for Linux, but I don't know much Windows.

It's convenient to know it for debugging purposes if you ever need to connect it through Bastion.
I can generate a random password.